src/compile/data.ts

```diff
@@ -650,6 +762,11 @@ export namespace stackScale {
     return null;
   }
 
+  export function parseLayer(model: LayerModel) {
+    // TODO
```

Comment:
https://github.com/vega/vega-lite/issues/1254

On Sat, Mar 19, 2016 at 4:32 PM Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In src/compile/data.ts
> <https://github.com/vega/vega-lite/pull/1241#discussion_r56756819>:
>
> > @@ -650,6 +762,11 @@ export namespace stackScale {
> >      return null;
> >    }
> >
> > +  export function parseLayer(model: LayerModel) {
> > +    // TODO
>
> Do we have an issue for this? If not, please file one in the follow-up
> milestone
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/1241/files/2d40fd567bcc48dcc87430cf20a363eb191f8d32#r56756819>
>